pkg/repositories/models/execution.go

```diff
@@ -34,16 +34,13 @@ type Execution struct {
 	// Prefixed with Execution to avoid clashes with gorm.Model UpdatedAt
 	ExecutionUpdatedAt *time.Time
 	Duration           time.Duration
-	ExecutionEvents    []ExecutionEvent
```

Comment:
why is this deleted? how do we store the relationship now?

Reply:
I don't find this field is being read anywhere . Please let me know how we are using it .

Reply:
ah cool i think you're right